pkg/acs/calacs/include/acs.h

```diff
@@ -13,6 +13,9 @@
 # define ACS_FITS_REC       82
 # define SZ_STRKWVAL        68
 
+#define ACS_WFC_N_COLUMNS_PER_CHIP_INCL_OVERSCAN 4144
```

Comment:
This should now be able to correct subarrays of any size irrespective of which chip they are on.
BTW there is only one subarray test in our regression suite and I already rigorously tested this (this time) across two macs and nott. It works fine. I also tested this on the fullframe images in the suite. I say this because once this is merged into ``candidate-1.3.0`` it'll be tagged and delivered to DMS.